README.md

```diff
@@ -4008,6 +4028,7 @@ Message Type: **response**
 |`emergencyEvent`|EmergencyEvent|False|Information related to an emergency event (and if it occurred)|
 |`clusterModeStatus`|ClusterModeStatus|False|The status modes of the cluster|
 |`myKey`|MyKey|False|Information related to the MyKey feature|
+|`windowStatus`|WindowStatus|False|Information related to window status|
```

Comment:
I think the description should indicate that this is an array of `WindowStatus` structs